libs/stocks/feature-shell/src/lib/stocks/stocks.component.html

```diff
@@ -29,7 +30,6 @@
     </mat-select>
   </mat-form-field>
 
-  <button (click)="fetchQuote()" mat-raised-button>Go</button>
```

Comment:
Removed the Go button as it is now redundant with fetchQuote() being called upon updating form information